File: department-service/src/main/java/com/dimageshare/department/entity/Department.java
package com.dimageshare.department.entity;

import lombok.Data;

import javax.persistence.*;

@Entity
@Data
@Table(name = "department")
public class Department {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    private String name;
    private String description;
}

File: python-microservices/department-service/com/entropy/entity/department.py
class Department(object):
    def __init__(self, description, name, id=None):
        self.description = description
        self.name = name
        self.id = id

    def get_id(self):
        return self.id

    def get_description(self):
        return self.description

    def get_name(self):
        return self.name